NsqSharp.Tests/Bus/AutofacBusTest.cs

```diff
@@ -66,7 +66,7 @@ public void AutofacContainerTest()
                         },
                         preCreateTopicsAndChannels: true);
 
-                BusService.Start(busConfiguration);
+                busConfiguration.StartBus();
```

Comment:
All these tests require this change now. ``BusService`` is not supported in NET Core/Standard